hybridse/java/hybridse-native/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
         <artifactId>hybridse-java</artifactId>
         <groupId>com.4paradigm.hybridse</groupId>
-        <version>0.2.4-SNAPSHOT</version>
+        <version>0.2.4.20210928-SNAPSHOT</version>
```

Comment:
This change should reverted before merge into main

Reply:
Could we just use single version in https://s01.oss.sonatype.org/content/repositories/snapshots/com/4paradigm/hybridse/hybridse-sdk/0.2.4.20210928-SNAPSHOT/ ? 
E.g 'hybridse-sdk-0.2.4.20210928-20210928.183823-1.jar'